rest-api/model/biobank_stored_sample.py

```diff
@@ -21,6 +21,8 @@ class BiobankStoredSample(Base):
   # The participant the sample is associated to. We use Biobank's ID for streamlined importing.
   biobankId = Column('biobank_id', Integer, ForeignKey('participant.biobank_id'))
 
+  # The biobank order identifier. to enable joining on reconciliation report.
+  biobankOrderIdentifier = Column('value', String(80), nullable=False)
```

Comment:
Let's call this column biobank_order_identifier. The "value" name for the column in biobank_order_identifier table only makes sense given the table name (and presence of "system" column in the same table.)

This will require you to alter or regenerate your schema migration.